cypress/integration/tests/developer/controls/stakeholders/sort.test.ts

```diff
@@ -79,9 +76,9 @@ describe("Stakeholder sort validations", { tags: "@tier2" }, function () {
         if (hasToBeSkipped("@tier2")) return;
 
         // Delete the job functions, stakeholder groups and stakeholders created before the tests
-        deleteAllJobfunctions();
-        deleteAllStakeholders();
-        deleteAllStakeholderGroups();
+        deleteByList(jobFunctionsList);
+        deleteByList(stakeholdersList);
+        deleteByList(stakeholderGroupList);
```

Comment:
Are these methods        deleteAllJobfunctions(), deleteAllStakeholders()
deleteAllStakeholderGroups() redundant then ?

Reply:
`deleteByList` was created because sometimes tests were failing when trying to remove item that was in use and that was created before current test and not related to it. 
Old functions still might be needed if you need to remove EVERYTHING and you don't have a list of what was created, I'd rather leave them where they are.

Reply:
As an option I can create a new method that will grab for you all existing items and push them into array, that will allow us to remove existing methods that you mentioned. In case when we will need to remove everything - we will grab all content of the table, push them to the list and just remove everything by that content. This can be done, I guess.